{"timestamp":"2025-10-29T05:51:51.137861","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T05:51:51.140164","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/airflow-dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T05:51:51.476156","level":"info","event":"Message populated: <Task(PythonOperator): populate_message>","logger":"airflow.task"}
{"timestamp":"2025-10-29T05:51:51.478248","level":"info","event":"Phrase extracted: <Task(PythonOperator): process_and_assemble_messages>","logger":"airflow.task"}
{"timestamp":"2025-10-29T05:51:51.479755","level":"info","event":"Submission sent.","logger":"airflow.task"}
{"timestamp":"2025-10-29T05:51:51.613503Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:51.614335Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:51.614839Z","level":"info","event":"Current task name:process_and_assemble_messages","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:51.615102Z","level":"info","event":"Dag name:sqs_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:51.506298","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-10-29T05:51:51.614893","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-10-29T05:51:53.519330","level":"error","event":"Error getting queue attributes: An error occurred (InvalidAddress) when calling the GetQueueAttributes operation: The address https://sqs.us-east-1.amazonaws.com/ is not valid for this endpoint.","logger":"airflow.task"}
{"timestamp":"2025-10-29T05:51:53.519844","level":"error","event":"Cannot form dataframe","logger":"airflow.task"}
{"timestamp":"2025-10-29T05:51:53.520258","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"InvalidAddress","exc_value":"An error occurred (InvalidAddress) when calling the GetQueueAttributes operation: The address https://sqs.us-east-1.amazonaws.com/ is not valid for this endpoint.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":129,"name":"assemble_message"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":108,"name":"assemble_message"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":58,"name":"get_queue_attributes"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":44,"name":"get_queue_attributes"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":569,"name":"_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":1023,"name":"_make_api_call"}]}]}
{"timestamp":"2025-10-29T05:51:53.522951Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:53.523373Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:53.523675Z","level":"info","event":"Task:<Task(PythonOperator): process_and_assemble_messages>","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T05:51:53.523932Z","level":"info","event":"Failure caused by An error occurred (InvalidAddress) when calling the GetQueueAttributes operation: The address https://sqs.us-east-1.amazonaws.com/ is not valid for this endpoint.","chan":"stdout","logger":"task"}
